elyra/templates/airflow/notebook_op.py

```diff
@@ -54,18 +51,19 @@ def __init__(self,
         self.cos_pull_archive = cos_pull_archive
         self.container_work_dir = "jupyter-work-dir"
         self.pipeline_inputs = pipeline_inputs
-        self.cos_endpoint = urlparse(cos_endpoint)
+        self.cos_endpoint = cos_endpoint
         self.cos_dir_pre = cos_directory + '/'
         self.pipeline_outputs = pipeline_outputs
-        self.dependencies = dependencies
+        self.bootstrap_script_url = bootstrap_script_url
 
         if self.bootstrap_script_url is None:
             """ If bootstrap_script arg with URL not provided, use the one baked in here.
             """
-            self.bootstrap_script_url = 'http://yukked1.fyre.ibm.com/bootstrapper.py'
+            self.bootstrap_script_url = 'https://raw.githubusercontent.com/elyra-ai/' \
+                                        'kfp-notebook/master/etc/docker-scripts/bootstrapper.py'
 
         if 'image' not in kwargs:  # default image used if none specified
-            kwargs['image'] = 'lresende/notebook-kubeflow-pipeline:dev'
+            kwargs['image'] = 'elyra/tensorflow:1.15.2-py3'
```

Comment:
Once Luciano's BYOImage PR lands, we'll probably want to way to somehow convey that a given image is "the default".  If they're loaded in a well-defined order, this could be as simple as taking the first in the list, or we could add a property that indicates the instance can be used as a default (but we'd likely need to enforce uniqueness across the instances (so only one is the default)). At any rate, nothing to do now, but we'll eventually need to pull this value from somewhere.